source/MRViewer/MRSceneCache.h

```diff
@@ -57,20 +57,20 @@ class MRVIEWER_CLASS SceneCache
 #pragma GCC diagnostic ignored "-Wstrict-aliasing"
```

Comment:
There is something bad in your code, if you ignore this.

Reply:
reworked